tutorials/pub-sub/README.md

```diff
@@ -503,9 +503,11 @@ kubectl logs --selector app=python-subscriber -c python-subscriber
 <!-- STEP
 name: Deploy Csharp App
 expected_stdout_lines:
-  - "A:  Message on A"
-  - "B:  Message on B"
-  - "C:  Message on C"
+  - "A: Message on A"
+  - "B: Message on B"
```

Comment:
Given there is a bit of a race and timing condition here, I suggest we put in a sleep to give each message more time to process and validate in order.

Reply:
@paulyuk I just noticed that the first csharp subscriber step already has the sleep on it but I have seen that sometimes is failing because of order, I think we should put sleep in places that we are sending the messages for each step?
Are those steps in the Readme also? is this one of them?
<!-- STEP
name: Curl validation k8s

After I got your confirmation I would apply the change and change the messages to use logger again and push it up.

Reply:
Yes I believe we should expand on sleeps to make it as deterministic as possible.  Perhaps we really increase sleeps like to 5 seconds to ensure?

Reply:
also agree it might be bette rto try where we're sending the messages.  Those all come from the same process and might be more deterministic.

Reply:
@paulyuk Sounds good, so Should I just revert the Console to logger and push it up or do I need to add sleep for steps also?

Reply:
Hey. I could be ok either way. What's your preference ?  My only thought is most other quickstarts use console

Reply:
Hey @paulyuk I prefer consistency, I fixed the whitespace issue after adding the Console and added sleep directive to C# subscriber step only. Please let me know what else I need to do?

Reply:
Ok I like the choice you made.  Does it pass well and "look good to you now"?  I will also test.

Reply:
It seems that it does, and I think it looks good, thanks for asking me :)

Reply:
Awesome. I will carve out some time to give it a test and final look, today hopefully.

Thank you so much for driving this.  Improving test stability has a lot of positive impact on the project.